File: __tests__/node_path_test.ml
open Jest

external join : string array -> string = "" [@@bs.module "path"]  [@@bs.splice]

let _ =

describe "join" (fun _ ->
  let open Expect in

  test "concat paths with sep" (fun _ ->
    expect @@ join [|"a"; "b"|] |> toEqual "a/b");
);

File: lib/js/__tests__/node_path_test.js
'use strict';

var Jest = require("@glennsl/bs-jest/lib/js/src/jest.js");
var Path = require("path");

describe("join", (function () {
        return Jest.test("concat paths with sep", (function () {
                      return Jest.Expect[/* toEqual */12]("a/b", Jest.Expect[/* expect */0](Path.join("a", "b")));
                    }));
      }));

/*  Not a pure module */
